### ISSUE TITLE
Better config validation

### ISSUE THREAD

**Issue by okonet** [ai/size-limit#20]
1. Providing an object instead of array in the config results in 

```
 ERROR  TypeError: limits.map is not a function
    at readPkg.then.result (/Users/okonet/Projects/OSS/attr-accept/node_modules/size-limit/cli.js:98:31)
    at <anonymous>
```

It would be much better to see a human friendly validation error saying that config must be an Array of objects with an example.

2. Running without config results in 

```
 ERROR  Can not find "size-limit" section in package.json.
        Add it according Size Limit docs.
```

Much better experience would be to generate the config on the first run automatically or at least help with an example.

**Comment by ai** [comment]
Agree. I will try to fix it today.

**Comment by ai** [comment]
Done 0739471

**Comment by ai** [comment]
I show example, since changing `package.json` doesn’t look like safe thing :).

**Comment by ai** [comment]
Made config check even smarter d2ad554

**Comment by ai** [comment]
Released in 0.11.2. Thanks for the idea.
